app/assets/javascripts/templates/page_overlay.es6.jsx

```diff
@@ -0,0 +1,50 @@
+class PageOverlay extends Component {
+
+  // --------------------------------------------------
+  // Setup
+  // --------------------------------------------------
+  constructor(props) {
+    super(props);
+    this._node = null;
+  }
+
+  // --------------------------------------------------
+  // Styles
+  // --------------------------------------------------
+  get styles() {
+    return {
+      container: {
+        display: 'flex',
+        justifyContent: 'center',
+        position: 'fixed',
+        top: '0px',
+        left: '0px',
+        zIndex: StyleConstants.planes.nine,
+        width: '100vw',
+        height: '100vh',
+        backgroundColor: StyleConstants.colors.fog,
+      },
+    };
+  }
+
+  get clickableStyles() {
+    return {
+      default: {
+        position: 'fixed',
+        top: '0px',
+        right: '0px',
+        zIndex: StyleConstants.planes.nine,
+        padding: '8px',
+      },
+    };
+  }
+
+  // --------------------------------------------------
+  // Lifecycle
+  // --------------------------------------------------
+  componentDidMount() {
+    var node = ReactDOM.findDOMNode(this.refs.container);
+    this._node = node;
+    node.addEventListener('click', (event) => this.handleClick(event));
+  }
```

Comment:
Can you create a placeholder `render` method in this component? The `Clickable` component has an example of this.